test/unit/plugin.test.js

```diff
@@ -0,0 +1,24 @@
+import {assert} from 'chai'
```

Comment:
Let's avoid using `unit` directory for test file.

Reply:
It is an existing folder and the current test config doesn't support it in the root `test` folder. What name do you suggest?

Reply:
@vinteo oh sorry, i think we update tests (migrate to `jest`) in this repo, do nothing, leave everything as is :+1: